runner-core/src/test/java/org/apache/apisix/plugin/runner/handler/A6ConfigHandlerTest.java

```diff
@@ -157,12 +153,11 @@ void testAddFilter3(CapturedOutput capturedOutput) {
 
         A6Conf config = cache.getIfPresent(0L);
         Assertions.assertEquals(config.getChain().getFilters().size(), 1);
-        Assertions.assertTrue(capturedOutput.getOut().contains("skip the same filter: FooFilter"));
     }
 
     @Test
     @DisplayName("test receive undefined filter")
-    void testAddFilter4(CapturedOutput capturedOutput) {
```

Comment:
Why not replace it with `System.setOut(new PrintStream(bytes));` like what you do in A6HttpCallHandlerTest.java?

Reply:
Not entirely possible, as these logs are defined in the source code. and the logging for A6HttpCallHandlerTest.java is defined in the test code. I will try to work it out in another way.